### ISSUE TITLE
Cannot read accessibility instructions.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#19977]
Testing -177674

![image](https://user-images.githubusercontent.com/3840081/226743733-9ac431ad-95e3-43fb-a9da-eb30449723d5.png)
